docs/configuration-reference.md

```diff
@@ -137,6 +138,37 @@ key | value | description
 :--|:--|:--
 field | string | The name of the field to drop.
 
+### drop_event
+
+The `drop_event` processor will remove all events where the specified field
+exactly matches one of the listed values.
+This can be used to filter datasets from ingress to service based on which
```

Comment:
Is this referring to the Kubernetes object kinds _Ingress_ and _Service_? If so, write them in distinguishing type; at minimum, capitalized, and ideally set in italic as well.